parsec/core/win_registry.py

```diff
@@ -74,22 +74,25 @@ def winreg_read_user_dword(key: str, name: str) -> object:
 def winreg_write_user_dword(key: str, name: str, value: int) -> None:
     winreg = get_winreg()
     hkcu = winreg.HKEY_CURRENT_USER
-    with winreg.OpenKey(hkcu, key, access=winreg.KEY_SET_VALUE) as key_obj:
+    with winreg.CreateKey(hkcu, key) as key_obj:
         winreg.SetValueEx(key_obj, name, None, winreg.REG_DWORD, value)
 
 
 def winreg_write_user_string(key: str, name: str, value: str) -> None:
     winreg = get_winreg()
     hkcu = winreg.HKEY_CURRENT_USER
-    with winreg.OpenKey(hkcu, key, access=winreg.KEY_SET_VALUE) as key_obj:
+    with winreg.CreateKey(hkcu, key) as key_obj:
         winreg.SetValueEx(key_obj, name, None, winreg.REG_SZ, value)
 
 
 def winreg_delete_user_dword(key: str, name: str) -> None:
     winreg = get_winreg()
     hkcu = winreg.HKEY_CURRENT_USER
-    with winreg.OpenKey(hkcu, key, access=winreg.KEY_SET_VALUE) as key_obj:
-        winreg.DeleteValue(key_obj, name)
+    try:
+        with winreg.OpenKey(hkcu, key, access=winreg.KEY_SET_VALUE) as key_obj:
+            winreg.DeleteValue(key_obj, name)
+    except OSError:
+        pass
```

Comment:
No need to log as info or warning to know when a key could not be deleted and remains in the registry?

Reply:
Right I wanted to catch `FileNotFound` errors when the key does not exist (hence nothing to delete). But this `except OSError` is probably too much of a catchall